google/cloud/spanner/options.h

```diff
@@ -84,6 +84,14 @@ using SpannerPolicyOptionList =
     OptionList<spanner::SpannerRetryPolicyOption, SpannerBackoffPolicyOption,
                SpannerPollingPolicyOption>;
 
+/**
+ * Option for `google::cloud::Options` to set the database role used for
+ * session creation.
```

Comment:
Consider a link to the spanner documentation explaining what these what these roles are for.

Reply:
I'm afraid I'm unaware of, and cannot discover, any released documentation.  We can, of course, add something as it becomes available.